source/adapters/level_zero/command_buffer.cpp

```diff
@@ -177,7 +177,7 @@ ur_result_t calculateKernelWorkDimensions(
             Device->ZeDeviceComputeProperties->maxGroupSizeX,
             Device->ZeDeviceComputeProperties->maxGroupSizeY,
             Device->ZeDeviceComputeProperties->maxGroupSizeZ};
-        GroupSize[I] = std::min(size_t(GroupSize[I]), GlobalWorkSize[I]);
```

Comment:
Why is this change to put `std::min` in parenthesis required?

Reply:
The build was failing on SYCL CI on Windows platform with an error `'min' : is not a member of 'std'` or `'(' : illegal token on right side of '::'`, I don't remember exactly anymore.

Reply:
This is a common problem because Windows defines `min` and `max` as macros and they collide with std functions, ref: https://stackoverflow.com/questions/11544073/how-do-i-deal-with-the-max-macro-in-windows-h-colliding-with-max-in-std

Reply:
oh wow, I've never come across that Windows issue before, thanks for the explanation.